CHANGES.md

```diff
@@ -60,6 +60,9 @@
 <!-- Major changes to documentation and policies. Small docs changes
      don't need a changelog entry. -->
 
+- Document the ignoring of `pyproject.toml` files without a `tool.black` section in
```

Comment:
I don't want this in the changelog; I feel that sets a precedent that we need to document changes to internal functions.

Reply:
I understand, I just included it because the changelog github action failed otherwise. Will remove it in a moment.